docs/Contributing/guides/integrations/README.md

```diff
@@ -0,0 +1,14 @@
+# Fleet Integration Guides
+
+This directory contains guides for integrating Fleet with external systems.
+
+## Integration Guides
+
+- [SCIM Integration](scim-integration.md) - Guide for integrating Fleet with SCIM
```

Comment:
Feels like these guides should be moved into the product group dirs, and this article dropped, since "integrations" is a broad enough term that there should be way more of these articles.

Reply:
Agreed this could be filled out more with article links. The reason I put this in the general guides is that these integrations don't fall under the purview of a single product group. Open to changing later.